android_tutorial_cv_bridge_CompressedImage_javacv/src/main/java/com/github/rosjava/android_extras/android_tutorial_cv_bridge/MainActivityCompressedJavacv.java

```diff
@@ -0,0 +1,148 @@
+package com.github.rosjava.android_extras.android_tutorial_cv_bridge;
```

Comment:
Before you merge this one, please put rosjava_extras on rosjava_mvn_repo.
then check if it compile,

Thanks.
For me it compile and run.